packages/stores/src/queries/pools/pools.ts

```diff
@@ -128,6 +145,9 @@ export class ObservableQueryPools extends ObservableChainQuery<Pools> {
     }
   );
 
+  /**
+   * @deprecated
+   */
```

Comment:
Since we haven't published, let's just remove instead of deprecate.

Reply:
This was intended to change before publishing. Put deprecated as a note for me to modify in the future.